mteb/tasks/Classification/multilingual/CataloniaTweetClassification.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification, MultilingualTask
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGS = {
+    "spanish": ["spa-Latn"],
+    "catalan": ["cat-Latn"],
+}
+
+
+class CataloniaTweetClassification(MultilingualTask, AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CataloniaTweetClassification",
+        description="""This dataset contains two corpora in Spanish and Catalan that consist of annotated Twitter
+        messages for automatic stance detection. The data was collected over 12 days during February and March
+        of 2019 from tweets posted in Barcelona, and during September of 2018 from tweets posted in the town of Terrassa, Catalonia.
+        Each corpus is annotated with three classes: AGAINST, FAVOR and NEUTRAL, which express the stance
+        towards the target - independence of Catalonia.
+        """,
+        reference="https://aclanthology.org/2020.lrec-1.171/",
+        dataset={
+            "path": "catalonia_independence",
+            "revision": "cf24d44e517efa534f048e5fc5981f399ed25bee",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train", "test"],
```

Comment:
Let's use validation instead of train. Same for n_samples and avg_character_length below.
```suggestion
        eval_splits=["validation", "test"],
```